src/UI/UIView.cpp

```diff
@@ -1000,18 +1025,18 @@ void UIView::save_user_session()
     
     if (m_use_transferfunction)
     {
-      save_trans_functions();
-      savefile << "Trnfnc " << m_dir_to_save + helper::get_OS_path_separator() + m_save_file_name + ".fnc" << "\n";
+      savefile << "Trnfncs" << "\n";
+      save_trans_functions(savefile);
```

Comment:
when saving a transfer function, if the same file name is selected will that overwrite or append to the file with the current logic?

Reply:
It will overwrite the file. Any thoughts?

Reply:
I think overwriting would be expected, so that's good in my book